src/app/code/community/Zendesk/Zendesk/Block/Adminhtml/Dashboard/Tab/Tickets/Grid/Massaction.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/**
+ * Copyright 2015 Zendesk
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+class Zendesk_Zendesk_Block_Adminhtml_Dashboard_Tab_Tickets_Grid_Massaction extends Mage_Adminhtml_Block_Widget_Grid_Massaction {
+
+    public function getJavaScript() {
+        return " {$this->getJsObjectName()} = new varienGridMassaction('{$this->getHtmlId()}', "
```

Comment:
Native magento grids do not support massactions. Such method appends required js to initialize that feature. 
http://prntscr.com/667oov